kcidb/db/sqlite/v04_04.py

```diff
@@ -0,0 +1,115 @@
+"""Kernel CI report database - SQLite schema v4.3"""
+
+import logging
+import kcidb.io as io
+from kcidb.misc import merge_dicts
+from kcidb.db.sqlite.schema import \
+    BoolColumn, TextColumn, TimestampColumn, JSONColumn, Table
+from .v04_03 import Schema as PreviousSchema
+
+# Module's logger
+LOGGER = logging.getLogger(__name__)
+
+
+class Schema(PreviousSchema):
+    """SQLite database schema v4.4"""
+
+    # The schema's version.
+    version = (4, 4)
+    # The I/O schema the database schema supports
+    io = io.schema.V4_5
+
+    # A map of table names and Table constructor arguments
+    # For use by descendants
+    TABLES_ARGS = merge_dicts(
+        PreviousSchema.TABLES_ARGS,
+        checkouts=merge_dicts(
+            PreviousSchema.TABLES_ARGS["checkouts"],
+            columns=dict(
+                {
+                    "git_commit_tags": JSONColumn(),
+                    "git_commit_message": TextColumn(),
+                    "git_repository_branch_tip": BoolColumn(),
+                }
+            )
```

Comment:
You're replacing *all* columns with the new ones here. That's why CI fails. See how `v04_03.py` does this.

Also you can run pytest locally to reproduce this.

Reply:
Thanks for the pointer.